shell/platform/fuchsia/dart_runner/dart_test_component_controller_v2.cc

```diff
@@ -521,7 +551,7 @@ bool DartTestComponentControllerV2::RunDartMain() {
     FX_LOGF(ERROR, LOG_TAG, "Failed to allocate Dart arguments list: %s",
             Dart_GetError(dart_arguments));
     Dart_ExitScope();
-    return false;
+    return fpromise::make_ok_promise();
```

Comment:
I'm unfamiliar with fpromise, but shouldn't these be `make_error_promise`s instead of `make_ok_promise`s, to match the behavior before where we were returning false?